src/templates/ru/2019/methodology.html

```diff
@@ -0,0 +1,432 @@
+          <img src="/static/images/methodology-banner.png" class="content-banner" alt="" width="1200" height="984" loading="eager" />
+        </picture>
+        <h2 id="overview"><a href="#overview" class="anchor-link">Краткий обзор</a></h2>
+
+        <p>
+          The Web Almanac — это проект, организованный <a href="https://httparchive.org">HTTP Archive</a>. HTTP Archive был запущен в 2010 Стивом Садерсом (Steve Souders), чтобы отслеживать развитие веба. Каждый месяц этот сервис анализирует содержимое миллионов веб-страниц и записывает терабайты метаданных в <a href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data">BigQuery</a> для последующего анализ. Подробнее <a href="https://httparchive.org/about">об HTTP Archive</a>.
+        </p>
+
+        <p>
+          Задача Web Almanac заключается в том, чтобы сделать хранилищем данных HTTP Archive более доступным и понятным для веб-сообщества при помощи профильных экспертов. Представляйте это себе как ежегодное хранилище общедоступных сведений о состоянии веба с 2019 года (первое издание отчёта).
+        </p>
+
+        <p>
+          Выпуск Web Almanac за 2019 года состоит из четырех разделов: контент, пользовательские ощущения, публикация и распространение. Каждая часть отчёта является одним из краеугольных камней и состоит из глав, в которых она рассматриваются с различных аспектов. Например, вторая часть представляет пользовательские ощущения и включает главы о производительности, безопасности, доступности, SEO, PWA и мобильном вебе.
+        </p>
+    </section>
+
+    <section>
+        <h2 id="dataset"><a href="#dataset" class="anchor-link">О наборе данных</a></h2>
+
+        <p>
+          Набор данных HTTP Archive постоянно растёт каждый месяц. Для издания Web Almanac 2019 года, если в главе не указано иное, все метрики были взяты из сканирования сайтов за июль 2019 года. Эти показатели можно <a href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">свободно запросить</a> из таблиц с префиксом <code>2019_07_01</code> на BigQuery.
+        </p>
+
+        <p>
+          Все метрики, представленные в Web Almanac, можно получить из набора данных на BigQuery. Вы можете посмотреть запросы, используемые во всех главах, в нашем <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2019">репозитории на GitHub</a>.
+        </p>
+
+        <p class="note">
+          Обратите внимание, что некоторые запросы могут быть достаточно большими и <a href="https://cloud.google.com/bigquery/pricing">дорогостоящими</a> для выполнения, так как оплата за пользование BigQuery начинается после обработки одного терабайта данных. Для того, чтобы помочь вам контролировать расходы, прочитайте пост Тима Кадлека (Tim Kadlec) <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.
```

Comment:
`lang="en"` attribute if showing English. Also should we translate the post name?

```suggestion
          Обратите внимание, что некоторые запросы могут быть достаточно большими и <a href="https://cloud.google.com/bigquery/pricing">дорогостоящими</a> для выполнения, так как оплата за пользование BigQuery начинается после обработки одного терабайта данных. Для того, чтобы помочь вам контролировать расходы, прочитайте пост Тима Кадлека (Tim Kadlec) <a lang="en" href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.
```

Reply:
I think it is necessary to leave the original post name, as there is no translation into Russian.

Reply:
Yeah was in two minds myself for that reason. Happy to resolve now you’ve added the lang attribute.